scripts/keeper.ts

```diff
@@ -43,14 +43,13 @@ async function getDomainsToFlush(
   );
   const logMessageEvents = await starknet.queryFilter(
     logMessageFilter,
-    findNearestBlock(l1Signer.provider, Date.now() - flushDelayMultiplier * flushDelay)
+    await findNearestBlock(l1Signer.provider, Date.now() - flushDelayMultiplier * flushDelay)
```

Comment:
We want recent flushes here. `flushDelayMultiplier` should be ~ 1. But `flushesToBeFinalized`  expects `flushesToBeFinalized` to be higher.

Reply:
I don't think we need flushDelayMultiplier anymore